static/js/stream_data.js

```diff
@@ -766,6 +774,17 @@ export function get_notifications_stream() {
     return "";
 }
 
+export function realm_has_reportmsg_stream() {
+    return page_params.realm_reportmsg_stream_id !== -1;
+}
+
+// reporting a message is a "blind" operation where the reporting user
+// doesn't have to be subscribed to the target stream. The stream
+// name is not important and none of that user's business!
+export function get_reportmsg_stream_id() {
+    return page_params.realm_reportmsg_stream_id;
+}
```

Comment:
This is unused.

Reply:
DONE - doh, sorry. thx.

(have we tried adding a linter that checks for unused defs?  if not, I might be game for trying it...)

Reply:
We do lint for unused definitions, but this isn’t unused from that perspective because it’s an export. It’s possible to lint for unused exports, but it’s much slower and results in a lot of false positives, mostly due to the silly `zrequire` syntax shortcut in the tests.